docs/draft/notifications.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+title: Notifications
+nav_order: 2
+has_children: true
+permalink: /specs/draft
+---
+
+# Notifications in Status
+
+## Why is there no Push Notifications?
```

Comment:
It'd be better to start by describing how notifications work (to the extent that they do). Then we can have this as an FAQ/motivation for now doing X.

I'd also cut some of the "earlier implementations..." as it isn't useful in a spec. If we want to keep this, it should be in a later section, like FAQ etc.